test/assert.js

```diff
@@ -2251,26 +2251,33 @@ describe('assert', function () {
         heroes = ['spiderman', 'superman'],
         fn     = function() { obj.value += 5 },
         fnDec  = function() { obj.value -= 20 },
+        getterFn = function() { return obj.value },
+        getter= function() {},
         bangFn = function() { obj.str += '!' },
         smFn   = function() { 'foo' + 'bar' },
         batFn  = function() { heroes.push('batman') },
         lenFn  = function() { return heroes.length };
 
+    getter.value= obj.value;
```

Comment:
Missing a space here too. But this line will probably end up being deleted due to never using `getter`.